### ISSUE TITLE
Add node ID to Viewer output

### ISSUE THREAD

**Issue by sjpb** [parmes/solfec#77]
For the `tools | point coordinates` or `tools | points distance` options in the Viewer it'd be really helpful to get the node numbers of the selected nodes output as well.

**Comment by tkoziara** [comment]
ok, thanks:) I'll have a look.

**Comment by tkoziara** [comment]
@sjpb I am looking at this again; would you be able to share about what is the context of application of solfec in your case? thank you, Tomek:)

**Comment by tkoziara** [comment]
@sjpb this feature has now been implemented as 699af0f; feel free to test and provide feedback if needed; example output looks now like this:
```
POINT: [0.6, 0.05, 0] (node 8)
POINT: [0.5, 0.05, 0.2] (node 16)
POINT: [1, 1, 0] (node 15)
DISTANCE from [0.6, 0.25, 0] (node 8) to [0.6, 0.65, 0] (node 8): 0.4
DISTANCE from [0.6, 0.65, 0.4] (node 26) to [1, 0, 0] (node 0): 0.861684
ANGLE between [0.6, 0.45, 0] (node 8) and [0.6, 0.2, 0] (node 2) and [1, 1, 0] (node 15): 143.973
ANGLE between [0.6, 0.65, 0] (node 8) and [0.6, 0.05, 0] (node 8) and [0.6, 0.65, 0.4] (node 26): 90
```
Note, that node numbers are local to individual body MESH or CONVEX based geometries. The type of underlying geometry is not specified in the printouts (it could be if it was of practical value). Closing the issue - do feel free to reopen it if required.
